<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qsampler - A LinuxSampler Qt GUI Interface.

   Copyright (C) 2004-2012, rncbc aka Rui Nuno Capela. All rights reserved.
   Copyright (C) 2007, Christian Schoenebeck

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

</comment>
 <class>qsamplerDeviceForm</class>
 <widget class="QDialog" name="qsamplerDeviceForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>378</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Qsampler: Devices</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qsampler.qrc" >:/images/qsamplerDevice.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter1" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth" >
      <number>4</number>
     </property>
     <widget class="QTreeWidget" name="DeviceListView" >
      <property name="minimumSize" >
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Device list</string>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights" >
       <bool>true</bool>
      </property>
      <column>
       <property name="text" >
        <string>Devices</string>
       </property>
      </column>
     </widget>
     <widget class="QSplitter" name="splitter2" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth" >
       <number>4</number>
      </property>
      <widget class="QWidget" name="DeviceParamLayout" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="DeviceNameTextLabel" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip" >
             <string>Device name</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="indent" >
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="DriverNameTextLabel" >
            <property name="text" >
             <string>Dri&amp;ver:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>DriverNameComboBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="DriverNameComboBox" >
            <property name="toolTip" >
             <string>Driver type name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="DeviceParamTable" >
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PortParamLayout" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="DevicePortTextLabel" >
            <property name="text" >
             <string>Channel:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>DevicePortComboBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="DevicePortComboBox" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Device port/channel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="DevicePortParamTable" >
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="RefreshDevicesPushButton" >
       <property name="toolTip" >
        <string>Refresh device list view</string>
       </property>
       <property name="text" >
        <string>&amp;Refresh</string>
       </property>
       <property name="icon" >
        <iconset resource="qsampler.qrc" >:/images/formRefresh.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+R</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CreateDevicePushButton" >
       <property name="toolTip" >
        <string>Create device</string>
       </property>
       <property name="text" >
        <string>&amp;Create</string>
       </property>
       <property name="icon" >
        <iconset resource="qsampler.qrc" >:/images/deviceCreate.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteDevicePushButton" >
       <property name="toolTip" >
        <string>Delete device</string>
       </property>
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
       <property name="icon" >
        <iconset resource="qsampler.qrc" >:/images/deviceDelete.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClosePushButton" >
       <property name="toolTip" >
        <string>Close this dialog</string>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="icon" >
        <iconset resource="qsampler.qrc" >:/images/formReject.png</iconset>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <tabstops>
  <tabstop>DeviceListView</tabstop>
  <tabstop>RefreshDevicesPushButton</tabstop>
  <tabstop>DriverNameComboBox</tabstop>
  <tabstop>DevicePortComboBox</tabstop>
  <tabstop>CreateDevicePushButton</tabstop>
  <tabstop>DeleteDevicePushButton</tabstop>
  <tabstop>ClosePushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qsampler.qrc" />
 </resources>
 <connections/>
</ui>
